src/Servers/Kestrel/Core/src/Internal/Http3/Http3FrameWriter.cs

```diff
@@ -21,6 +21,10 @@ namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http3
 {
     internal class Http3FrameWriter
     {
+        // Size based on HTTP/2 default frame size
+        private const int MaxDataFrameSize = 16 * 1024;
+        private const int HeaderBufferSize = 16 * 1024;
```

Comment:
Should these be configurable?

Does msquic have any frame size limits?

Reply:
I don't know about how msquic writes data. There is no max frame size in HTTP/3 so I don't know if a limit is required at all, or whether it is best to write data in as large as chunks as possible.

I don't see a reason to make them configurable at the moment. Can add settings later if required.